src/utils/connection.ts

```
@@ -56,6 +56,10 @@ export function buildRequiredNamespaces(networks: ('ethereum' | 'solana' | 'bsc'
           break;
         case 'base':
           chains.push(CHAIN_IDS.BASE_MAINNET, CHAIN_IDS.BASE_SEPOLIA);
+        case 'polygon':
+          chains.push(CHAIN_IDS.POLYGON_MAINNET, CHAIN_IDS.POLYGON_AMOY);
```

Biome (2.1.2)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)